NextcloudTalk/CallViewController.h

```diff
@@ -61,6 +61,8 @@
 @property (nonatomic, strong) IBOutlet UILabel *callTimeLabel;
 @property (nonatomic, strong) IBOutlet UIView *screenshareLabelContainer;
 @property (nonatomic, strong) IBOutlet UILabel *screenshareLabel;
+@property (weak, nonatomic) IBOutlet UIView *participantsLabelContainer;
+@property (weak, nonatomic) IBOutlet UILabel *participantsLabel;
 
```

Comment:
Any reason to have those as weak instead of strong as the other outlets?

Reply:
Made them strong